src/components/Tables/Public.tsx

```diff
@@ -87,6 +110,13 @@ export default async function PublicTable({
           </tbody>
         </table>
       </div>
+      {
+        totalPages > 0 
+          ? <Pagination totalPages={ totalPages } /> 
+          : (<div className="text-center py-5">
+            <h2 className="text-xl"> Oops, we have nothing like that in our database...</h2>
```

Comment:
Now I show this message if no public study is searched. May need to check with Toby.